### ISSUE TITLE
Null pointer exceptions if content it not found

### ISSUE THREAD

**Issue by wanze** [riasvdv/statamic-linkit#9]
Hi,

If we link to non-existing content, a null pointer exception is thrown causing to break the site.
This can be fixed by checking if the content is existing prior to call its `url()` method:

```php
       // In LinkitModifier::getUrl() 

       if (isset($link['asset'])) {
            $asset = Asset::find($link['asset'][0]);
            $url = $asset ? $asset->url() : '';
        }

        if (isset($link['term'])) {
            $term = Term::find($link['term'][0]);
            $url = $term ? $term->url() : '';
        }

        if (isset($link['page'])) {
            $page = Page::find($link['page'][0]);
            $url = $page ? $page->url() : '';
        }

        if (isset($link['entry'])) {
            $entry = Entry::find($link['entry'][0]);
            $url = $entry ? $entry->url() : '';
        }
```

Thanks for creating this addon! 👍 

Cheers

**Comment by riasvdv** [comment]
Good catch! I'll fix this as soon as I can

**Comment by riasvdv** [comment]
Fixed in 1.1.1
